Solvers/solvePDE.m

```diff
@@ -1,7 +1,9 @@
-function phi = solvePDE(MS, M, RHS, varargin)
-%SOLVEPDE solves the linear system M x \phi = RHS and returns the value of
-%\phi reshaped based on the structure of the MeshStructure variable. The
-%default solver is the matlab '\' linear solver
+function phi = solvePDE(MS, M, RHS, guess, solver_opts)
```

Comment:
I think this one is a breaking change. I prefer to keep the old `solvePDE` function and use `varargin` to introduce the new options (with default values). It is not as clean as this one but I guess we have to accept it since Matlab does not have multiple dispatch.